java-grupa8/homework-1/tema/src/main/java8/ro/siit/tema/SalesDepartement.java

```diff
@@ -0,0 +1,40 @@
+package ro.siit.tema;
+
+/**
+ * @author x-RauL-x
+ * A class that holds the information about the sales departaments
+ */
+public class SalesDepartement {
```

Comment:
This code is part of another task(homework). It should not show up in this pull request.